src/xenia/kernel/xboxkrnl/xboxkrnl_memory.cc

```diff
@@ -425,8 +425,14 @@ DECLARE_XBOXKRNL_EXPORT2(MmQueryAddressProtect, kMemory, kImplemented,
 
 void MmSetAddressProtect_entry(lpvoid_t base_address, dword_t region_size,
                                dword_t protect_bits) {
-  if (!protect_bits) {
-    XELOGE("MmSetAddressProtect: Failed due to incorrect protect_bits");
+  constexpr uint32_t required_protect_bits =
+      X_PAGE_NOACCESS | X_PAGE_READONLY | X_PAGE_READWRITE |
+      X_PAGE_EXECUTE_READ | X_PAGE_EXECUTE_READWRITE;
+
+  if (xe::bit_count(protect_bits & required_protect_bits) != 1) {
+    // Many titles use invalid combination with zero valid bits set.
```

Comment:
Can you please provide the title IDs and the bit values passed by those titles (and, if needed, reproduction steps) in the code comments so it's easier to look up the reason and regression test cases in the future (without having to search for the pull request, and without depending on a specific location of the repository on the web)?

Reply:
Added